master/internal/command/authz_basic_impl.go

```diff
@@ -42,6 +45,40 @@ func (a *NSCAuthZBasic) CanSetNSCsPriority(
 	return nil
 }
 
+// CanGetTensorboard returns true and nil error unless the developer master config option
+// security.authz._strict_ntsc_enabled is true then it returns a boolean if the user is
+// an admin or if the user owns the tensorboard and a nil error.
+func (a *NSCAuthZBasic) CanGetTensorboard(
+	ctx context.Context, curUser *model.User, tb *tensorboardv1.Tensorboard,
+) (canGetTensorboards bool, serverError error) {
+	if !config.GetMasterConfig().Security.AuthZ.StrictNTSCEnabled {
```

Comment:
since https://github.com/determined-ai/determined/pull/5639  has landed I don't think we want this logic here anymore. just return nil?

Reply:
Ah right, good catch

Reply:
did you mean to remove this?